.bandit

```diff
@@ -0,0 +1,10 @@
+[bandit]
```

Comment:
@JJRR5 Please remove the pre-commit-vauxoo files from the commits, they must not be in it. And after that you add those files on the `.gitignore` so they are ignored when trying to commit.